en/simulation/flightgear_vehicles.md

```diff
@@ -121,9 +121,9 @@ The file content meaning is as follows:
 * `FgModel` - a precise case sensitive name of the FlightGear model corresponding to "XXXX-set.xml" in the model directory (where XXXX is the model name).
 * `Url` is optional and it is not currently used. It is intended for future use to auto-download the models from web
 * `Controls` - the most important part of the process of adding a vehicle. This section contains the mapping between the PX4 mixer file and [FlightGear property tree](http://wiki.flightgear.org/Property_tree).
-  * The first number in a list selects a PX4 mixer output.
+  * The first number in a list selects a PX4 mixer output. 
```

Comment:
Advice on what to do with this? It might be completely orthogonal to our control allocation stuff.

Reply:
Yes it's the counterpart to the allocation on the simulator side. Essentially it just means they need to match.

Reply:
OK. I think this can be just left alone. Thanks.